src/components/pages/CompanyData/components/FormFields.tsx

```diff
@@ -92,11 +92,23 @@ const UpdateForm = ({
       <div style={{ marginTop: '34px' }}>
         <ValidatingInput
           name="siteName"
-          label={t('content.companyData.site.form.site.name')}
+          label={
+            isAddress
+              ? t('content.companyData.address.form.site.name')
+              : t('content.companyData.site.form.site.name')
```

Comment:
I'd suggest to write all those more compact as

`label={t('content.companyData.${isAddress ? 'address' : 'site'}.form.site.name')}`

(replace outer ' with ` )

Reply:
updated